packages/yourloops/lib/patient/patient.util.ts

```diff
@@ -94,33 +94,36 @@ export default class PatientUtils {
     })
   }
 
-  static isInAtLeastATeam = (patient: Patient): boolean => {
-    const tm = patient.teams.find((team: PatientTeam) => team.status === UserInvitationStatus.accepted)
-    return !!tm
-  }
-
   static isInvitationPending = (patient: Patient): boolean => {
     const tm = patient.teams.find((team: PatientTeam) => team.status === UserInvitationStatus.pending)
     return !!tm
   }
 
-  static isOnlyPendingInvitation = (patient: Patient): boolean => {
-    const tm = patient.teams.find((team: PatientTeam) => team.status !== UserInvitationStatus.pending)
-    return !tm
+  static getAllPatients = (patients: Patient[]): Patient[] => {
```

Comment:
```suggestion
  static getNonPendingPatient = (patients: Patient[]): Patient[] => {
```
Seems like a better name, don't you think?

Reply:
I'm not sure, maybe we could have more categories in the future (ex: archived which was planned). I agree AllPatients isn't quite accuracy, maybe `getCurrentPatients` ?